config/sync/user.role.contributor.yml

```diff
@@ -49,21 +49,16 @@ permissions:
   - 'access user profiles'
   - 'add citation entities'
   - 'add content to books'
-  - 'administer main menu items'
   - 'break content lock'
   - 'create file media'
   - 'create google_form media'
   - 'create image media'
   - 'create media'
   - 'create policy log'
-  - 'create stanford_course content'
   - 'create stanford_event content'
-  - 'create stanford_event_series content'
   - 'create stanford_gallery_images media'
   - 'create stanford_news content'
   - 'create stanford_page content'
-  - 'create stanford_person content'
```

Comment:
I think we may want to leave this. Large site owners, it turns out, sometimes use student workers to add alumni profiles, etc.